boards/arm/nrf9161dk_nrf9161/revision.cmake

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+board_check_revision(
+  FORMAT MAJOR.MINOR.PATCH
+  DEFAULT_REVISION 0.7.1
```

Comment:
This should be the version that is actually sent to customers. Not some old prototype version.

Reply:
Thanks, updated to the latest 0.9.1